newIDE/app/src/MainFrame/index.js

```diff
@@ -211,6 +213,7 @@ const MainFrame = (props: Props) => {
     openSubscriptionDialog,
   ] = React.useState<boolean>(false);
   const [exportDialogOpen, openExportDialog] = React.useState<boolean>(false);
+  const preferences = React.useContext(PreferencesContext);
 
   React.useEffect(() => {
```

Comment:
Reminder: This has to be removed if it isn't needed anymore :)

Reply:
Yep! I'll remove this soon if we don't find any issues.

Reply:
Thanks for the reminder ;)